frontend/src/app/testResults/uploads/Uploads.tsx

```diff
@@ -208,32 +208,50 @@ const Uploads = () => {
             Report results in bulk using a comma-separated values (CSV)
             spreadsheet. To upload your spreadsheet:
           </p>
-          <ol className="usa-list">
-            <li>
-              <a
-                href="/assets/resources/test_results_example_10-3-2022.csv"
-                onClick={() => {
-                  appInsights?.trackEvent({
-                    name: "Download spreadsheet template",
-                  });
-                }}
-              >
-                Download the spreadsheet template
-              </a>{" "}
-            </li>
-            <li>
-              Format and add your data following the template and upload guide
-            </li>
-            <li>Save your spreadsheet as a CSV</li>
-            <li>Upload your CSV here</li>
-          </ol>
-          <p>
-            For more information about preparing your spreadsheet, visit our{" "}
-            <LinkWithQuery to="/results/upload/submit/guide">
-              <strong>spreadsheet upload guide</strong>
-            </LinkWithQuery>
-            .
-          </p>
+          <section>
+            <ol className="usa-process-list">
+              <li className="usa-process-list__item margin-bottom-1em">
+                <h2 className="usa-process-list__heading">
+                  Visit the{" "}
+                  <LinkWithQuery to="/results/upload/submit/guide">
+                    <strong>spreadsheet upload guide</strong>
+                  </LinkWithQuery>
+                </h2>
+              </li>
+              <li className="usa-process-list__item margin-bottom-1em">
+                <h2 className="usa-process-list__heading">
+                  Download the{" "}
+                  <a
+                    href="/assets/resources/test_results_example_10-3-2022.csv"
+                    onClick={() => {
+                      appInsights?.trackEvent({
+                        name: "Download spreadsheet template",
+                      });
+                    }}
+                  >
+                    spreadsheet template
+                  </a>
+                </h2>
+              </li>
+              <li className="usa-process-list__item margin-bottom-1em">
+                <h2 className="usa-process-list__heading">
+                  Following the guide and template, format your data to match
+                  SimpleReport requirements
+                </h2>
+              </li>
+              <li className="usa-process-list__item margin-bottom-1em">
+                <h2 className="usa-process-list__heading">
+                  Save your spreadsheet as a CSV
```

Comment:
@lauraykerr should this read as `CSV file` instead of `CSV`?

Reply:
Thanks for pointing this out @zdeveloper — can we actually say `Save your spreadsheet in a CSV format` to be consistent with steps in the guide?

Reply:
@zdeveloper @lauraykerr aye aye 🏴‍☠️ 

changed in [2837d12](https://github.com/CDCgov/prime-simplereport/pull/4619/commits/2837d124089d524e5b817c7dc967601dce7c2cb0).